scripts/pull_legacy_assets.py

```diff
@@ -5,7 +5,7 @@
 
 # the commented out links are not valid or cant be found
 agencies_links = {
-    "DoA": "https://usda.gov/code.json",
+    "USDA": "https://usda.gov/code.json",
```

Comment:
Because I made this change, we'll need to rerun the scripts. I'll message you guys tmr for your help @sachin-panayil @IsaacMilarky